src/libopensc/card-openpgp.c

```diff
@@ -272,6 +273,7 @@ static pgp_do_info_t	pgp34_objects[] = {	/**** OpenPGP card spec 3.4 ****/
 	{ DO_ENCR_SYM, SIMPLE,      READ_ALWAYS | WRITE_PIN3,  pgp_get_pubkey_pem, NULL   },
 	{ 0, 0, 0, NULL, NULL },
 };
+// clang-format on
```

Comment:
please look at the formatting report for a single remark in this file:
https://github.com/OpenSC/OpenSC/actions/runs/7479501161/job/20356826376?pr=2977

Reply:
Resolved the outstanding issues.